app/api/common.js

```diff
@@ -4,12 +4,12 @@
 
 import BigNumber from 'bignumber.js';
 import { defineMessages } from 'react-intl';
-import type { Features } from 'trezor-connect';
 
 import LocalizableError from '../i18n/LocalizableError';
 import WalletTransaction from '../domain/WalletTransaction';
 import WalletAddress from '../domain/WalletAddress';
 import Wallet from '../domain/Wallet';
+import type { WalletHardwareInfo } from '../types/WalletType';
```

Comment:
`WalletHardwareInfo` is a little misleading because it holds information that Ledger doesn’t have